### ISSUE TITLE
Please throw exception instead of dying

### ISSUE THREAD

**Issue by dmitryplatonov** [aewallin/openvoronoi#28]
It's very inconvenient when algorithm kilsl program instead of reporting failure.

**Comment by aewallin** [comment]
yes, this is a good idea. would you suggest using Boost Exception?
did you encounter a failed assert() or an uncontrolled crash?
